google/cloud/bigquery_storage_v1beta2/types/__init__.py

```diff
@@ -33,6 +33,7 @@
     TableFieldSchema,
 )
 from .stream import (
+    DataFormat,
```

Comment:
I think `DataFormat` is doubled up because these files didn't actually get regenerated. Should be fixed by https://github.com/googleapis/python-bigquery-storage/pull/117/files#diff-296a39f9422dac004121c56b959022e9ad234263312c6433f8c58e2e6cd0b61eR25